cypress/support/e2e.js

```diff
@@ -128,6 +128,8 @@ Cypress.Commands.add("takeSnapshots", (options = {}) => {
             title: forcedColorsActive ? `${Cypress.currentTest.titlePath.join(" ")} in high contrast` : undefined,
         };
 
+        cy.get("#toggle-screenshot-mode").trigger("click", { force: true });
```

Comment:
Hvis du tar inn forslaget mitt må det også endres her og på linje 143 😊

```suggestion
        cy.get("#screenshot-mode-toggle").trigger("click", { force: true });
```